app/assets/stylesheets/govuk_publishing_components/components/_print-link.scss

```
@@ -0,0 +1,52 @@
+.gem-c-print-link {
+  display: none;
+  margin-bottom: 2em;
+  margin-top: 2em;
+}
+
+.gem-c-print-link.gem-c-print-link__show-without-js {
```

```suggestion
.gem-c-print-link--show-without-js {
```
This class, being used at the block level, feels more like a modifier class than a different element.

---

fixed!